.travis.yml

```diff
@@ -6,9 +6,7 @@ php:
   - 7.3
 
 before_install:
-  - composer self-update
   - composer install
-  - composer require php-coveralls/php-coveralls
```

Comment:
I do not want to install php-coveralls on my local machine. I think it is useful only [on GitHub](https://github.com/php-coveralls/php-coveralls#prerequisites). I prefer that stay here.